components/BUILD.yaml

```diff
@@ -227,8 +227,20 @@ scripts:
         query="update d_b_user set blocked=0 where name=\"$user\";"
         mysql -e "$query" -u$DB_USERNAME -p$DB_PASSWORD -h 127.0.0.1 gitpod
       fi
+
+      printf "\nFGA: Reset fgaRelationshipsVersion for $user...\n"
+      query="update d_b_user set additionalData = JSON_REMOVE(additionalData, '$.fgaRelationshipsVersion') where name=\"$user\";"
+      mysql -e "$query" -u$DB_USERNAME -p$DB_PASSWORD -h 127.0.0.1 gitpod
+      if [ $? -ne 0 ]; then
+        echo "error resetting fgaRelationshipsVersion"
+        kill $PID || true
+        exit 1
+      fi
       kill $PID || true
 
+      printf "\nRestarting server and redis to pick up DB changes...\n"
+      kubectl rollout restart deployment/server deployment/redis
```

Comment:
I wonder why we decide to put such longer bash scripts into the leeway yaml? Woudln't it be better to have such scripts written in bash files somewhere and jsut called here (if that is at all necessary)?

Reply:
:shrug: I don't know. There's not much advantages I can think of, and some downsides.